modules/geolocation/unittests.py

```diff
@@ -117,7 +117,75 @@ def test_camera_offset_from_origin_pointing_sideways_with_not_enough_downward_pi
         # Test
         np.testing.assert_equal(actual, expected)
 
+    def test_identity_mapping(self):
+        # Setup
+        locator = geolocation.Geolocation()
+        locator._Geolocation__pixelToGeoPairs = np.array([[[1, 1], [1, 1]],
+                                                          [[-1, -1], [-1, -1]],
+                                                          [[1, -1], [1, -1]],
+                                                          [[-1, 1], [-1, 1]]])
 
-if __name__ == "__main__":
+        expected = np.array(np.eye(3))
+
+        # Run
+        actual = locator.calculate_pixel_to_geo_mapping()
+
+        # Test
+        np.testing.assert_almost_equal(actual, expected)
+
+    def test_rotation_90(self):
```

Comment:
Personally I would prefer two lines between class methods even if [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines) says to do one. The reason is that there are no brackets, so single-line blanks inside functions and single-blank lines between functions look very similar.